sources/packages/backend/apps/api/src/route-controllers/atbc/atbc.students.controller.ts

```diff
@@ -20,7 +16,7 @@ import { DISABILITY_REQUEST_NOT_ALLOWED } from "../../constants";
 
 @AllowAuthorizedParty(AuthorizedParties.student)
 @RequiresStudentAccount()
-@Controller("atbc")
+//@Controller("atbc") // TODO: This is commented as part of #2539 - Suspend any ATBC integration.
```

Comment:
As mentioned in https://github.com/bcgov/SIMS/pull/2604/files#r1428580193

I would say decorator can remain as it is.